harvester_e2e_tests/templates/vm_with_disk_cdrom.json.j2

```diff
@@ -0,0 +1,165 @@
+{# Template for basic VM creation request. Start the VM instance upon #}
```

Comment:
I don't think we need this template. The basic_vm template already have a cdrom device. See https://github.com/harvester/tests/blob/main/harvester_e2e_tests/templates/basic_vm.json.j2#L100